forge-gui/res/cardsfolder/upcoming/riku_of_many_paths.txt

```diff
@@ -0,0 +1,16 @@
+Name:Riku of Many Paths
+ManaCost:G U R
+Types:Legendary Creature Human Wizard
+PT:3/3
+T:Mode$ SpellCast | ValidActivatingPlayer$ You | ValidSA$ Spell.Modal | Execute$ TrigCharm | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a modal spell, ABILITY
+SVar:TrigCharm:DB$ Charm | ChoiceDesc$ up to X, where X is the number of times you chose a mode for that spell | Choices$ DBExile,DBPutCounter,DBToken | CharmNum$ TriggeredSpellAbility$NumTimesChoseMode | MinCharmNum$ 0
```

Comment:
I think if you'd use X together with AdditionalDescription$ this would build almost clean without need for another new param?

Reply:
I did look into that a bit
I think with the splashy complex "choose this many" it could be nice to have something like this
Or make the new param just pull the string between "choose" and "–" from the oracle line 😆